packages/docusaurus-init/templates/bootstrap/blog/2018-09-11-Towards-Docusaurus-2.md

```diff
@@ -0,0 +1,134 @@
+---
```

Comment:
We should keep templates light and easy to tell it's a template. Come up with something else?